### DISCUSSION TITLE
【恶意修改】版权标识的界限？

### DISCUSSION THREAD

**Discussion by fred913** [bs-community/blessing-skin-server#446] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/41846047/179696541-51f77b60-9e67-4bdf-b4ec-7ba8444e3a51.png)
上图中“（包括不限于删除、修改作者信息、修改链接指向）”中的“包括但不限于”可能不是特别清楚。在后台写了一段JavaScript代码，如下：
```javascript
$("#copyright-text").html('<a href="https://github.com/bs-community/blessing-skin-server" target="_blank">Blessing Skin Server</a> is the best.')
```
代码加载后footer如图所示：
![image](https://user-images.githubusercontent.com/41846047/179696852-595752bc-816a-4439-abc9-bb496791cdd6.png)
href未修改，想请问这种修改是否违反Blessing Skin Server版权规定？

**Comment by Big-Cake-jpg** [comment, marked as answer]
你不应该在 Discussion 里提问。建议打开一个 Issue 或者加入 Telegram 群组。
